amoro-format-iceberg/pom.xml

```diff
@@ -134,6 +134,11 @@
             <artifactId>parquet-avro</artifactId>
         </dependency>
 
+        <dependency>
```

Comment:
Do we need to declare it a dependency here, as `amoro-format-iceberg` has already dependent on `amoro-common`?

Reply:
+1. the parquet-jackson and parquet-avro are included

Reply:
Yes, so we may drop the dependencies here?